src/autoscaler/api/config/config.go

```diff
@@ -193,9 +195,9 @@ func loadVcapConfig(conf *Config, vcapReader configutil.VCAPConfigurationReader)
 		return err
 	}
 
-	configureEventGenerator(conf)
-	configureScheduler(conf)
-	configureScalingEngine(conf)
+	conf.ScalingEngine.TLSClientCerts = vcapReader.GetInstanceTLSCerts()
```

Comment:
putting it in a variable and using that variable looks better (avoid repetittion)

Reply:
fixed in 30d445128